### ISSUE TITLE
Picasa API Deprecation

### ISSUE THREAD

**Issue by twsiyuan** [tgulacsi/picago#10]
From the [Picasa document](https://developers.google.com/picasa-web/docs/3.0/deprecation), the API has been deprecated and is no longer available.

It's time to make a small change to makes this library still works until 15 March 2019, and sunset this project after that day.

**Comment by tgulacsi** [comment]
Right. This has been only a pilot project for Picasa / Google Photos integration into Camlistore (now Perkeep), and there we've been moved on with Drive.

Thanks for the reminder!
